index.html

```diff
@@ -43,33 +48,61 @@
+<section class="hallelujah">
     
-    <section class="Works">
-        <div class="Project-name"><h2>Projects</h2></div>
+    <section class="works">
+        <div class="project-name"><h2>Projects</h2></div>
         <div class="work one">
-            <div class="project photo"><img src="assets/work/work_one.png" class="work_one" alt="#"></div>
-            <div class="project summary">
-                <div class="nameP"><h3>Project name goes here</h3></div>
-                <div class="technology">
-                <ul class="hello">
-                    <div class="tech1"><li class="maybe"><p>HTML/CSS</p></li></div>
-                    <div class="tech1"><li class="maybe"><p>Ruby on rails</p></li></div>
-                   <div class="tech1"><li class="maybe"><p>JavaScript</p></li></div>
+            <div class="project photo"><img src="assets/work/work_one.png" class="work-one" alt="#"></div>
+            <div class="project summary desksum">
+                <div class="namep namedesk"><h3>Project name goes here</h3></div>
+                <div class="technology techdesk">
+                <ul class="hello hellodesk">
+                    <div class="tech1 tech1desk"><li class="maybe"><p>HTML/CSS</p></li></div>
+                    <div class="tech1 tech1desk"><li class="maybe"><p>Ruby on rails</p></li></div>
+                   <div class="tech1 tech1desk"><li class="maybe"><p>JavaScript</p></li></div>
                 </ul>
                 </div>
-                <div alt="#" class="project-view">
-                    <div class="word"><p>See this project</p></div>
-                    <div class="union"><img src="assets/work/Union.png" class="arrow" alt="#"></div>
+                <div alt="#" class="project-view viewdesk">
+                    <div class="word worddesk"><p>See this project</p></div>
+                    <div class="union uniondesk"><img src="assets/work/Union.png" class="arrow arrowdesk" alt="#"></div>
                 </div>
             </div>
         </div>
         
 
          <div class="work two">
-             <div class="project photo"><img src="assets/work/work_two.png" class="work_one" alt="#"></div>
+            <div class="option left"></div>
+            <div class="option middle"></div>
+            <div class="option right">
+                <div class="man">
+                    <h6>Project name goes...</h6>
+                    
+                       <ul class="hellso">
+                           <div class="tech2"><li class="maybe"><p>HTML/CSS</p></li></div>
+                               <div class="tech2"><li class="maybe"><p>Ruby on rails</p></li></div>
+                                  <div class="tech2"><li class="maybe"><p>+1</p></li></div>
+                       </ul>
+
+           
+                    <div claas="being"> 
+                        <div alt="#" class="project-view viewdesk">
+                            <div class="word worddesk"><p>See this project</p></div>
```

Comment:
Please ensure that the `See this project ->` should be a button that takes you to the Live Demo of the featured project or to the GitHub repository.